client/lib/posts/actions.js

```diff
@@ -215,7 +215,7 @@ export const saveEdited = options => async ( dispatch, getState ) => {
 	// when toggling editor modes, it is possible for the post to be dirty
 	// even though the content hasn't changed. To avoid a confusing UX
 	// let's just pass the content through and save it anyway
-	if ( ! changedAttributes.content && rawContent !== initialRawContent ) {
+	if ( ! get( changedAttributes, 'content' ) && rawContent !== initialRawContent ) {
```

Comment:
looks good. Did tickle an old thought though, maybe we should turn on https://www.npmjs.com/package/babel-plugin-transform-optional-chaining ?

Reply:
I would totally love to use optional chaining. But last time I was reading TC39 minutes, I learned there is a risk that the proposal will be totally reworked from scratch with a different syntax.

Reply:
that makes me sad. :(